contents/blog/equity_explainer.md

```diff
@@ -0,0 +1,50 @@
+
+The most common vesting schedule for tech companies these days is the 4-year/1-year cliff, which means that a quarter of your options vest after your 1-year work anniversary, and 1/36th of your remaining options vest each month for the following three years. As a company, we feel that this structure strikes a fair balance between incentivizing team members to stay and contribute to the growth of the company, while not locking them into “golden handcuffs” where they are required to put in 4 years of work to reap any of the benefits of being a part of a start-up.
+
+### But how exactly do share options turn you into a millionaire? 
+
+When you are granted your options on the first day of employment, the price you will pay to exercise them will stay static at whatever the Fair Market Value (FMV) per share is on your first day, irrespective of what the value is when you sell your purchased shares. The FMV is decided through an external, very complicated process known as a 409A, and the initial sale price of the share is usually set when the company “goes public” or “IPOs” by being listed on a public stock exchange like the NYSE. This is known as a “liquidation event” and in the simplest terms, the difference between your exercise price and the price of the share you sell in the public market is cash in your pocket. The reality of this is much more complicated, as there can be a number of [tax implications](https://secfi.com/learn/exercise-stock-options-tax-implications) depending on the type of option you were granted, when you decide to exercise them relative to when you sell them, and your overall income; but you really should talk to an accountant before making any serious decisions so you don’t end up with an unexpectedly massive tax bill.
+
+Making matters more complicated, the majority of standard terms in stock option plans limit your freedom in when and how you can exercise your options. 
+
+### At PostHog, we try to do things differently:
+
+|                           | **PostHog**                 | **The Other Guys**           |
+|---------------------------|-----------------------------|------------------------------|
+| **Vesting Start:**        | On your first day           | After a "probationary period |
+| **Exercise Window:**      | 10 _years_ after departure  | 90 _days_ after departure    |
+| **Merger or Acquisition** | Double-Trigger acceleration | No acceleration              |
+
+### Vesting Start:
```

Comment:
we should be upfront about the cliff too (we're the same as everyone else here), but it is worth making sure we're clear about it